gmime/part.go

```diff
@@ -103,6 +103,14 @@ func (p *Part) Headers() textproto.MIMEHeader {
 	return nil
 }
 
+// GetContentID returns the content ID (if there is one) per mime part
+// we expect all attachments to have a content-id
```

Comment:
there is also isattachment function if this is only expected on attachments https://github.com/sendgrid/go-gmime/pull/33/files#diff-e39577e7183b8f756ececf21707723e28ea58a5f06aa522703acd6e4552ab0dbR33

Reply:
hmmm, thats a good point.

This is how we implement it on our side.  I think its purpose should be generic and always check if it's requested on the `part`.

I'll update the description to reflect that.